crates/bevy_winit/src/winit_windows.rs

```diff
@@ -110,6 +110,9 @@ impl WinitWindows {
                     panic!("Cannot find element: {}.", selector);
                 }
             }
+
+            winit_window_builder =
+                winit_window_builder.with_prevent_default(window.prevent_default_event_handling)
```

Comment:
Should this value be possible to change during runtime?

Reply:
Do the winit apis allow us to?

Reply:
Good question. I'm not sure, I need to check.

Reply:
doesn't seem to be!